docs/3.x/tech/tutorials/exporting-lists.adoc

```diff
@@ -0,0 +1,35 @@
+---
+layout: doc
+title: Exporting Lists and Configurations
+description: Learn how to export your lists and Configurations in CleanSpeak
+---
+
+
+== Exporting Lists
+Note: Making changes to your lists after exporting will cause issues when importing.
+
+
+You may be asked by our support team to export your lists and configurations so that we can test it on our end.
+Maybe you're moving lists between systems, or you're doing an upgrade and you want to backup your CleanSpeak DB.
+Here we will show you how to quickly and easily export your lists and configurations in CleanSpeak.
+
+
+From the moderator dashboard you will click on the filter drop down found in the left navigation.
+
+image::admin-dashboard.png[Exporting Lists,width=1200]
+
+image::filter-dropdown.png[Exporting Lists, width=1200]
+
+At the very bottom of the drop down you will find the lists button.
+
+Here we see our Blacklist and kids chat list.
```

Comment:
Anytime you reference a panel or proper name of a feature, you should match what the user will see on the panel. 

> Here we see our Blacklist and Kids Chat ...  

In this case, this is a summary of the filter configuration, so you could say `... see our Blacklist and Kids Chat list summary.`.